torch/nn/parallel/distributed.py

```diff
@@ -1221,6 +1244,19 @@ def join_device(self):
     def join_process_group(self):
         return self.process_group
 
+    def _register_buffer_comm_hook(
+        self,
+        state,
+        hook: callable,
```

Comment:
nit: make state after hook? following up (func, args) convention?

Reply:
Thanks for pointing this out!

I followed the same order as comm hook (state, hook): https://github.com/pytorch/pytorch/blob/master/torch/nn/parallel/distributed.py#L1224.

Should we try to keep it the same or change the order here?